mempager/pager_test.go

```diff
@@ -96,3 +96,19 @@ func Test_Pager_CantReplacePageBufferPointer(t *testing.T) {
 	buf = checkpage.Buffer()
 	assert.Equal(t, make([]byte, DEFAULT_PAGE_SIZE), *buf)
 }
+
+func Benchmark_PagerGrowPages100(b *testing.B) {
+	benchmarkPagerPageGrowth(100, b)
+}
+
+func Benchmark_PagerGrowPages100000(b *testing.B) {
+	benchmarkPagerPageGrowth(100000, b)
+}
+
+func benchmarkPagerPageGrowth(i int, b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		pgr := NewPager(0)
+		pgr.GetOrAlloc(1)
```

Comment:
why have this line? why not just `pgr.GetOrAlloc(i)`?

Reply:
Good question, it's not entirely necessary knowing how the pages grow, but I liked the idea of being explicit allocation pages with some large gap in between, otherwise if we only allocate page `i` and the implementation changes underneath to optimize for a single lone allocation but not the sparse idea then we won't spot the regression